src/fmu/ensemble/realizationcombination.py

```diff
@@ -4,6 +4,9 @@
 from __future__ import division
 from __future__ import print_function
 
+import fnmatch
+import six
+
 import pandas as pd
```

Comment:
Same as in ensemblecombination.py (though here a blank has been added after six)
```suggestion
import fnmatch

import six
import pandas as pd
```